charts/synthetic-monitoring-agent/templates/deployment.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   labels:
     {{- include "synthetic-monitoring-agent.labels" . | nindent 4 }}
 spec:
+  {{- with .Values.deploymentStrategy }}
+  strategy:
+    {{- toYaml . | trim | nindent 4 }}
```

Comment:
```suggestion
    {{- toYaml . | nindent 4 }}
```

Reply:
Any reason to remove it? 
All other helm charts seen to have this if I'm correct.

Reply:
could you tell me any reason to include it ? any trailing whitespaces would already be trimmed.

Reply:
As mentioned, i copied it from other helm charts in this repo which all use the same.

Also, the `{{-` will not trim the input from value it self if I'm correct.

Reply:
Yeah but the CI lint test won't pass if there are trailing spaces in the `deployment strategy` value, so you don't have to worry about that.

Reply:
Ok, I'm fine with removing it, i just wanted it to match the rest of all the helm charts.

Reply:
Done